dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -13,15 +13,23 @@ do
 done
```

Comment:
Since you do both ROSA and --provider aws for OSD using the same script, I wonder if we should change this to maanged_aws or something to avoid confusion, since you are clearly doing more than ROSA within this script.

Reply:
correct, I feel too. Either keep both rosa and osd under one umbrella `managed_aws` or keep rosa and osd_aws as separate platform. I can re-work accordingly. 
@dry923 @morenod what are your thoughts ?

Reply:
I think merging them is fine TBH. They are virtually identical and we really don't even test straight OSD on AWS because we already do ROSA. Assuming that, I'd probably say we stick with ROSA since that is what most of the metadata will be tag'd with since we pull it from the system under test. Open to other opinions though.

Reply:
I am okay to keep them under ROSA for now unless @morenod has other opinions and work on it later if this cause any confusion.

Reply:
After checking all the code, I dont see too much shared code. I see a lot of conditionals like if [[ $INSTALL_METHOD == "osd" ]]; then, including all the code related to OSD and after that, the else containing the existing code for rosa.

After checking that, Im 75% on creating different installation scripts.

Its ok for this PR, but we should think on refactor when we have some time

Reply:
Yeah I am okay to create a new script as a different platform(`awsms`) itself, but currently there are real switch-over only at 5 places(install, post-install, cleanup and 2 get commands) in `rosa.sh` to run `ocm create` cli commands instead of `rosa create` cli and rests are just env sets and logs so i thought of sharing the same script.  
Other changes you see in the PR are adding new json file for OSD AWS and `hypershift.sh` changes for OSD.